src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/ccd/CcdClient.java

```diff
@@ -28,13 +29,13 @@ public CcdClient(
         this.authenticatorFactory = authenticatorFactory;
     }
 
-    public void completeAwaitingDcnProcessing(
+    public void completeDcnProcessing(
         String exceptionRecordCcdId,
         String service,
         String jurisdiction
     ) {
         log.info(
-            "Completing awaiting payment DCN processing. Exception record ID: {}, service: {}",
+            "Completing payment DCN processing. Exception record ID: {}, service: {}",
```

Comment:
This is completion of the waiting for DCN processing, not the completion of DCN processing. This change shouldn't be included.

Reply:
Yes, just followed the previous comment, will revert